test/unit/modules/utils/cookie-spec.js

```diff
@@ -112,10 +112,14 @@ describe('Keen.utils.cookie', function() {
 
   });
 
-  describe('.enabled', function(){
+  describe('enabled property', function(){
 
-    it('should return a boolean value', function(){
-      assert.isBoolean(this.cookie.enabled());
+    it('should be a boolean value by default', function(){
+      assert.isBoolean(this.cookie.enabled);
```

Comment:
Yes it would be. If it was a function it would simply return `this.enabled`, so I figured it's likely not required. Thoughts?